src/components/course-listing/course-card.jsx

```
@@ -7,8 +7,11 @@ import {
   CardFooter,
   CardHeader,
 } from "@/components/ui/card";
+import { useNavigate } from "react-router-dom";
 
 export default function CourseCard({ course, viewMode }) {
+  const navigate = useNavigate();
```

ESLint

[error] 'navigate' is assigned a value but never used.

(no-unused-vars)